### ISSUE TITLE
Views for Guild/Quest dashboard aggregates

### ISSUE THREAD

**Issue by maparent** [topicquests/sensecraft#159]
The guild/quest tables give a few aggregate values:

* number of active members of guild
* number of of players of quest
* number of of guilds playing quest
* number of of published moves

Right now the FE calculates this using memberships. It would be better to have a view with joins and grouping to calculate the counts.
It would also  be a great opportunity to make the selects more efficient. (public quests plus private quests of which I'm a member would be better implemented as a union in the view's select than a row-level security calculation.)

Basically it means adding the views in quests_functions/guilds_functions, then using them in the store fetch functions. (Some adjustments to types also.)

TODO: think about where to store scoring aggregates.

**Comment by maparent** [comment]
Implemented for quests on a branch. Need to choose what to show.
